src/main/java/com/maxmind/geoip2/record/RepresentedCountry.java

```diff
@@ -13,8 +17,15 @@
  * </p>
  */
 public final class RepresentedCountry extends Country {
-    @JsonProperty
-    private String type;
+
+    private final String type;
+
+    public RepresentedCountry(@JsonProperty("names") HashMap<String, String> names, @JsonProperty("geoname_id") Integer geoNameId,
```

Comment:
Copy paste mistake. I meant to make them all Map, but didn't do that in all places.